protocol/protobuf/application_metadata_message.proto

```diff
@@ -61,5 +61,6 @@ message ApplicationMetadataMessage {
     SYNC_WALLET_ACCOUNT = 45;
     ACCEPT_CONTACT_REQUEST = 46;
     RETRACT_CONTACT_REQUEST = 47;
+    SYNC_COMMUNITY_SETTINGS = 49;
```

Comment:
There's another PR pending that introduces the `48` field here, so I already went with `49`